components/impls/core/src/main/javascript/components/NewMapComponent/amd/engines/google/MapEngineGoogle.js

```diff
@@ -685,7 +685,11 @@ define([
       }
       this.map.setZoom(zoomLevel);
       if (!this.zoomExtends()) {
-        this.map.panTo(new google.maps.LatLng(38, -9));
+        if(!isNaN(centerLatitude ) && !isNaN(centerLongitude)){
+          this.map.panTo(new google.maps.LatLng(centerLatitude, centerLongitude));
+        } else {
+          this.map.panTo(new google.maps.LatLng(38, -9));
```

Comment:
What are these magic coordinates?
I think you should add a comment explaining what they are.

Reply:
Oh, they were there already...